<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogOnDialog</class>
 <widget class="QDialog" name="LogOnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log On To Database</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log On</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>300</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>43</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Name / IP Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="server_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>63</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>93</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>113</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="database_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>163</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>143</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>193</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>213</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>263</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>243</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logon_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>313</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Log On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>313</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>313</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/lm2/help_button.png</normaloff>:/plugins/lm2/help_button.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>343</y>
     <width>300</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>server_edit</tabstop>
  <tabstop>port_edit</tabstop>
  <tabstop>database_edit</tabstop>
  <tabstop>user_edit</tabstop>
  <tabstop>password_edit</tabstop>
  <tabstop>logon_button</tabstop>
  <tabstop>help_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
